LibAddonMenu-2.0/LibAddonMenu-2.0/LibAddonMenu-2.0.lua

```diff
@@ -349,9 +349,9 @@ local localization = {
         TRANSLATION = "Translation",
         DONATION = "Donate",
         PANEL_INFO_FONT = "$(CHAT_FONT)|14|soft-shadow-thin",
-        RELOAD_UI_WARNING = "Changes to this setting require an UI reload in order to take effect.",
-        RELOAD_DIALOG_TITLE = "UI Reload required",
-        RELOAD_DIALOG_TEXT = "Some changes require an UI reload in order to take effect. Do you want to reload now or discard the changes?",
+        RELOAD_UI_WARNING = GetString(SI_OPTIONS_APPLY_WARNING),
```

Comment:
I just took a look at it ingame and you may be right. It feels strange to read "may result in a loading screen" when it does require one for sure. Please revert these changes and just leave the original text. Otherwise we'd have to update all the translations too. We can think about a better text for LAM3 in a separate ticket.

Reply:
Fair enough. I feel the same. I wonder if there's an easy way to do this with GitHub, without redoing the whole pull req.

Reply:
Just create another commit with the original text. I will squash it on merge anyways, so you won't see the individual commits, but only the end result.